Wireframe/index.html

```diff
@@ -6,27 +6,50 @@
     <title>Wireframe</title>
     <link rel="stylesheet" href="style.css" />
   </head>
+
   <body>
     <header>
+      <div>
       <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
+      <p class="titp"> Using the provided wireframe and resources, I'm writing a new webpage explaining </p>
+      </div>
     </header>
     <main>
-      <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+      <div class="div1">
+      <article class="article1">
+        <img src="1.png" alt="" />
```

Comment:
Please, add description to all your images inside of `alt` attribute. The description should be short and explain what is on the picture (for users who can't see).